ExampleMod/Common/Players/ExampleRegenPlayer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.Players
+{
+	public class ExampleRegenPlayer : ModPlayer
+	{
+		public override ModPlayer.Regen ModifyLifeRegen() {
+			var modifiers = ModPlayer.Regen.Create(); // Creates the struct with all values configured to do no modifications.
+			modifiers.deltaRate = 3f; // Increases the rate at which player reaches maximum natural life regeneration for every multiple of 300 up to 3600. Positives for buffs, negatives for debuffs.
+			modifiers.deltaRegen = 6f; // Increases the life per second recovered by typically 6 / 2 per second. Positives for buffs, negatives for debuffs.
+			modifiers.multiplyRegen = -0.2f; // Decreases effective regen to 20% potency and negates it. Typically positive and >1 for strong buffs, and negative and closer to zero for weak debuffs.
```

Comment:
negative multipliers seem like a fundamentally bad idea... since if two mods both apply them....

Reply:
This is an interesting one. 
We allowed this before, so I see no reason not to allow it now.
If both apply a negative, then they cancel each other ( and would have done so previously anyway).
I've updated the example to not use negatives, but just warn in the comment about what negatives in the field would do, mentioning both mod stacking and inversion. Some mod may want a debuff that inverts damaging debuffs and healing natural regen (ie some new player race type thing?)